e2e/testcafe-devextreme/tests/scheduler/timezones/globalConfigTimezoneDB.ts

```diff
@@ -0,0 +1,92 @@
+import Scheduler from 'devextreme-testcafe-models/scheduler';
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { ClientFunction } from 'testcafe';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`Timezone DB backward compatibility`
+  .page(url(__dirname, '../../container.html'));
+
+const setGlobalConfig = ClientFunction(() => {
+  (window as any).DevExpress.config({
+    timezones: [
+      {
+        id: 'Europe/London', untils: 'Infinity', offsets: '5', offsetIndices: '0',
+      },
+    ],
+  });
+});
+
+const data = [
```

Comment:
Looks like that some of these appointments shouldn't be visible on screenshot
Let's delete redundant ones to make this data set simplier